static/js/notifications.js

```diff
@@ -303,9 +309,12 @@ function process_notification(notification) {
                                " other people";
         }
         title += " (to you and " + other_recipients + ")";
+        raw_operators = [{operand: message.reply_to, operator: "pm-with"}];
     }
     if (message.type === "stream") {
         title += " (to " + message.stream + " > " + message.subject + ")";
+        raw_operators = [{operand: message.stream, operator: "stream"}];
+        if (message.subject !== "(no topic)") {raw_operators[1] = {operand: message.subject, operator: "topic"};}
```

Comment:
Oh, I just excluded `(no topic)` naturally. I just thought: "Oh, no topic, so no narrowing to a topic". Now that I actually think about it, `(no topic)` is also an actual topic. Maybe I should change this.